### ISSUE TITLE
Label is announced twice for the Icons displayed on the alerts

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4583]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.1.1 Non-text Content

**Test Environment:**
- Test URL: https://clarity.design/documentation/alerts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to any of the icons (info, warning, error etc) provided for different types of alerts.

**Expected result:**
Icon labels should be announced appropriately i.e. label should be announced along with the info that the element is image/graphics.

**Actual result:**
Label is announced twice. Refer to the below example:
Error icons are announced as "Error, graphic, error".

**Root cause:**
Title attribute and aria-labelledby attribute is used for the previous/next graphic icons. Values of those attributes are same, so screen reader is announcing those values.

**Reporter Email:** gkhadke@vmware.com

**Comment by gkhadK** [comment]
Similar issue is observed for Icon buttons from buttons or buttons group component.

**Comment by chlane** [comment]
We need to decide if SVG's inside the icons provide meaning or if they should be hidden. See the examples below.

[Compliant Code Example]

<svg role="img" focusable="false" aria-labelledby="svgTitle" aria-describedby="svgDescription"> <title id="svgTitle">Search</title>
 <desc id="svgDescription">This is a description of the search</desc>
 ...
</svg>


<svg role="none" focusable="false" aria-hidden="true"> ...
</svg>

**Comment by chlane** [comment]
Additionally, the title attribute creates a mouse-only tooltip that cannot be accessed by sighted keyboard-only users. We recommend to remove the title attribute from all elements. If a tooltip must be implemented, refer to https://confluence.eng.vmware.com/display/A11Y/ARIA+Tooltips.
